curvefs/src/metaserver/storage/rocksdb_storage.h

```diff
@@ -141,7 +113,7 @@ class RocksDBStorage : public KVStorage, public StorageTransaction {
 
     STORAGE_TYPE Type() override;
 
-    bool GetStatistics(StorageStatistics* Statistics) override;
+    // bool GetStatistics(StorageStatistics* Statistics) override;
```

Comment:
Why move away every `StorageStatistics`？

Reply:
it's not proper to put it in here, because we have multiple database instance.